concourse/steps/update_component_deps.py

```diff
@@ -410,6 +572,32 @@ def create_upgrade_pr(
     )
     from_component = from_component_descriptor.component
 
+    to_component_descriptor = component_descriptor_lookup(
+        ocm.ComponentIdentity(
+            name=to_ref.componentName,
+            version=to_version,
+        )
+    )
+
+    to_component = to_component_descriptor.component
+
+    cfg_factory = ctx.cfg_factory()
+    cfg_set = cfg_factory.cfg_set(ci.util.current_config_set_name())
+    delivery_endpoints_cfg = cfg_set.delivery_endpoints()
+
+    delivery_base_url = delivery_endpoints_cfg.dashboard_url()
```

Comment:
as the getter-method suggests, this is the delivery-dashboard-url. so I suggest to call the variable such, to make the name more expressive, and self-explanatory.